app/models/application_record.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  mount_uploader :avatar, AvatarUploader
+  validates :username, :firstname, :lastname, :email, :bio, :presence => true
```

Comment:
_PR_ stands for _Pull Request_, which is the mechanism we're using to review, and ultimately "pull", the changes you've "requested" we merge :)

Line 3 makes sense here in this PR because it is enabling the upload functionality for the model, and therefore fits with the rest of the logical changes for adding/enabling avatar upload functionality. 

Regarding line 4, my suggestion is to move that change into a new branch and make a new PR focused on adding any required validations for the Member model. 

Speaking of which... as I was just looking for the name of the Member model, I noticed that you have added these changes directly to `ApplicationRecord`. This class is inherited by every model in the application. I don't think every model in the app will have an avatar, or username, etc. :) 

> Where should I move line 3 and 4?

Both lines 3 and 4 should ultimately live in the `Member` model.